tools/qt-builder/README.md

```diff
@@ -160,60 +140,80 @@ Upload qt5-install-5.12.3-windows.tar.gz to our Amazon S3 vircadia-public bucket
+~~`git apply --ignore-space-change --ignore-whitespace patches/aec.patch`~~
+~~`git apply --ignore-space-change --ignore-whitespace patches/qtscript-crash-fix.patch`~~
+~~`cd ..`~~
 
-???`../qt5/configure -opensource -confirm-license -gdb-index -nomake examples -nomake tests -skip qttranslations -skip qtserialport -skip qt3d -skip qtlocation -skip qtwayland -skip qtsensors -skip qtgamepad -skip qtspeech -skip qtcharts -skip qtmacextras -skip qtvirtualkeyboard -skip qtpurchasing -skip qtdatavis3d -no-warnings-are-errors -no-pch -prefix ../qt5-install`  
-#### Make  
-`make`  
+#### Configuring
+```bash
+mkdir qt5-install
+mkdir qt5-build 
+cd qt5-build
+```
+
+amd64:
+```bash
+../qt5/configure -force-debug-info -release -opensource -confirm-license -platform linux-g++-64 -recheck-all -nomake tests -nomake examples -skip qttranslations -skip qtserialport -skip qt3d -skip qtlocation -skip qtwayland -skip qtsensors -skip qtgamepad -skip qtcharts -skip qtx11extras -skip qtmacextras -skip qtvirtualkeyboard -skip qtpurchasing -skip qtdatavis3d -skip qtlottie -skip qtquick3d -skip qtpim -skip qtdocgallery -no-warnings-are-errors -no-pch -no-egl -no-icu -prefix ../qt5-install
+```
+If libX11 or libGL aren't found, you will need to manually provide those locations. Search for `libX11.so` and `libGL.so` respectively and provide paths to their folders inside `qt5/qtbase/mkdspecs/linux-g++-64/qmake.conf`.
+On Ubuntu 18.04 both are in `/usr/lib/x86_64-linux-gnu`
+
+aarch64:
+```bash
+../qt5/configure -force-debug-info -release -opensource -confirm-license -platform linux-g++ -recheck-all -nomake tests -nomake examples -skip qttranslations -skip qtserialport -skip qt3d -skip qtlocation -skip qtwayland -skip qtsensors -skip qtgamepad -skip qtcharts -skip qtx11extras -skip qtmacextras -skip qtvirtualkeyboard -skip qtpurchasing -skip qtdatavis3d -skip qtlottie -skip qtquick3d -skip qtpim -skip qtdocgallery -no-warnings-are-errors -no-pch -no-icu -prefix ../qt5-install
+```
+You can accelerate the build process by installing some of the optional system dependencies.
 
-????*Ubuntu 18.04 only*  
-????`make module-qtwebengine`  
-????`make module-qtscript`  
+#### Make
+Replace `4` with the number of threads you want to use. Keep in mind that the QT build process uses a lot of memory. It is recommended to have at *least* 1,2 GiB per thread.
+Be warned that the qtwebengine build process ignores this setting and just uses your actual number of available threads instead. In my case it needed ~24GiB of memory for the build process.
+On some systems qtscript, qtwebengine, qtwebchannel and qtspeech will be build by `make` without you manually specifying it, while others require you to build modules individually.
+Check the folders of each needed module for binary files, to make sure that they have actually been built.
+```bash
+make -j4
+make -j4 module-qtscript
+make -j4 module-qtwebengine
+make -j4 module-qtspeech
+make -j4 module-qtwebchannel
+make -j4 install
```

Comment:
Is the sequence of individual `make $component` necessary?

Number of cores for qtwebengine can be set with `NINJAFLAGS=-j4` (or similar)

Reply:
On the last build it wasn't necessary. On every build before that it was necessary. It is also necessary on aarch64. When building it seems necessary to check the folders of the modules for binary files and make them individually if needed.
Maybe I am missing something here, but at the very least I wouldn't know what.

I will look at the NINJAFLAGS

Reply:
I will rearrange the module making information as well.